### PR TITLE
Login logout and welcome message banner

### PR DESCRIPTION
**Pull request by kavin1711**
1)added code for news letter and welcome message banner.
 2)updated support and config files
3)Removed before and after hooks

### CODE REVIEW

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  cy.acceptCookieBanner();
+
+  // Click the login button to navigate to the login page
+  cy.get(loginButton).click();
+
+  // Wait for 2 seconds for the login page to fully load
+  cy.wait(2000);
+
+  // Enter invalid credentials
+  cy.get(creds).eq(0).type("testing@gmail.com");
```
Comment (npriya21) [inline TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js:177]: All the const values or common values , might be good in fixtures or const files

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  cy.get(creds).eq(1).type("1234");
+
+  // Click the login button to submit the login form
+  cy.get(loginSubmitButton).click();
+
+  // Verify if an error message is displayed for incorrect credentials
+  cy.xpath(
+    "//div[@class='auth0-global-message auth0-global-message-error']//span[@class='animated fadeInUp']"
```
Comment (npriya21) [inline TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js:185]: Dont hard code the xpath

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/welcomeBanner.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+const environment = Cypress.env("ENV") || "prod";
+const url = Cypress.env(`${environment}_url`);
+
+/**
+ * Validates the welcome banner behavior for a guest user.
+ */
+export function welcomeBannerGuest() {
+  // Visit the homepage and accept the cookie banner
+  cy.visit(url, { timeout: 20000 });
+  cy.acceptCookieBanner();
```
Comment (npriya21) [inline TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/welcomeBanner.js:25]: Instead calling every fucntions, better to add in before each

**Review by npriya21 — CHANGES_REQUESTED** [review]
Minor review comments are added

File: TLS_E2E_Automation_Framework/cypress.config.js
Comment (cchiremath123) [inline TLS_E2E_Automation_Framework/cypress.config.js:1]: don't commit cypress.config file which is already there in current running repo

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/welcomeBanner.js (hunk 3 above)
Comment (kavin1711) [reply TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/welcomeBanner.js]: updated

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js (hunk 2 above)
Comment (kavin1711) [reply TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js]: updated

File: TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js (hunk 1 above)
Comment (npriya21) [reply TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js:161]: else can create const value on top

Comment (kavin1711) [reply TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/loginLogout.js]: updated

**Review by npriya21 — APPROVED** [review]
Approved

**Pull request merged**